akka-stream/src/main/resources/reference.conf

```diff
@@ -2,6 +2,8 @@
 # Akka Stream Reference Config File #
 #####################################
 
+# eager creation of the system wide materializer
+akka.library-extensions += "akka.stream.SystemMaterializer"
```

Comment:
Should it really be eager given that it will not always be used (existing code will likely rather create their own actormaterializer)?

Reply:
Agreed let's remove this